# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/manager/ssl/nsNSSIOLayer.cpp
# Commit: 20f3d7a5481d
# Full Hash: 20f3d7a5481db59c20e27a1603a58a40fdad9c93
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-02-10 21:33:47
# Regressor Bug: 1607194
# File Overlap Count: 1
# Description:
#   Bug 1607194 - P1: NSS initialization without nsNSSComponent for  the socket process r=keeler
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D58774
# ==============================================================================

diff -r 79b7b94df8b6 -r 20f3d7a5481d security/manager/ssl/nsNSSIOLayer.cpp
--- a/security/manager/ssl/nsNSSIOLayer.cpp	Mon Feb 10 15:53:23 2020 +0000
+++ b/security/manager/ssl/nsNSSIOLayer.cpp	Mon Feb 10 17:07:26 2020 +0000
@@ -1464,7 +1464,7 @@
 }
 
 void nsSSLIOLayerHelpers::loadVersionFallbackLimit() {
-  // see nsNSSComponent::setEnabledTLSVersions for pref handling rules
+  // see nsNSSComponent::SetEnabledTLSVersions for pref handling rules
   uint32_t limit = 3;  // TLS 1.2
 
   if (NS_IsMainThread()) {
